### ISSUE TITLE
Add Contributing Guidelines

### ISSUE THREAD

**Issue by matthewsot** [UC-Davis-CS-Tutoring/Handbook#1]
This can just link to the CONTRIBUTING.md file in Athena once it is merged into master. See UC-Davis-CS-Tutoring/Athena#2

This would supersede the "Want to Contribute?" message in the README, and provide a central place for contributors to find guidelines.

**Comment by matthewsot** [comment]
Closed as of PR #2
